bin/spotxBidAdapter.js

```diff
@@ -305,56 +316,7 @@ export const spec = {
 
 function outstreamRender(bid) {
   if (bid.renderer.config.outstreamFunction != null && typeof bid.renderer.config.outstreamFunction === 'function') {
-    bid.renderer.config.outstreamFunction();
-    return;
-  }
-
-  window.console.log('[SPOTX][renderer] Handle SpotX outstream/inbanner renderer');
```

Comment:
why did you remove this whole renderer section?

Reply:
Because it's smarter to keep that externally to the adaptor code in order to have different rendering possibilities by publisher. Thus, the idea is to share a sample prebid config file that is doing a classic rendering, and depending on their needs we can adapt the code client side without having to handle rendering option in a static adaptor code. Sorry if f I am not clear, you can check the implementation here http://testweb.spotx.tv/jmonbillard/spotx-prebid-goodAdaptor-outstream.html

Reply:
That's true, but we should leave this code in here, for pubs that don't want to have their own renderer. The way that prebid works, is if the publisher has registered a renderer on their side, then prebid will use that. But if that doesn't exist, it would use this

Reply:
http://prebid.org/dev-docs/show-outstream-video-ads.html#renderers

Reply:
I just pushed a new version including default renderer so publishers does not have to add it on their page. They can still choose to override the default one by a more custom one.